flight/Modules/Actuator/actuator.c

```diff
@@ -341,7 +420,7 @@ static void post_process_scale_and_commit(float *motor_vect,
 	 */
 	float maxpoweradd_softlimit = MAX(
 			2 * actuatorSettings.LowPowerStabilizationMaxPowerAdd,
-			desired_vect[MIXERSETTINGS_MIXER1VECTOR_THROTTLECURVE1])
+			fabsf(desired_vect[MIXERSETTINGS_MIXER1VECTOR_THROTTLECURVE1]))
```

Comment:
The clip management logic is simplified by doing everything as we're positive thrust, and then inverting it all at the end.